make/autoconf/util.m4

```diff
@@ -65,12 +65,11 @@ AC_DEFUN([UTIL_DEFUN_NAMED],
     ])
 
     m4_foreach([arg], [$3], [
-      m4_if(m4_bregexp(arg, [: ]), -1, m4_define([arg], m4_bpatsubst(arg, [:], [: ])))
-      m4_define(arg_name, m4_substr(arg, 0, m4_bregexp(arg, [: ])))
+      m4_define(arg_name, m4_substr(arg, 0, m4_bregexp(arg, [:])))
       m4_set_contains(legal_named_args, arg_name, [],[AC_MSG_ERROR([Internal error: m4_if(arg_name, , arg, arg_name) is not a valid named argument to [$1]. Valid arguments are 'm4_set_contents(defined_args, [ ]) m4_set_contents(legal_named_args, [ ])'.])])
       m4_set_remove(required_named_args, arg_name)
       m4_set_remove(legal_named_args, arg_name)
-      m4_pushdef([ARG_][]arg_name, m4_bpatsubst(m4_substr(arg, m4_incr(m4_incr(m4_bregexp(arg, [: ])))), [^\s*], []))
+      m4_pushdef([ARG_][]arg_name, m4_bpatsubst(m4_substr(arg, m4_incr(m4_bregexp(arg, [:]))), [^[[:space:]]*], []))
```

Comment:
On FreeBSD, with these changes, I get the following error trying to configure:

```
configure: error: unrecognized options: --disable-ccache, --disable-javac-server
configure exiting with result code 1
```

Reply:
Thanks for testing it. This has turned into a bit of a rabbit hole. It seems neither gnu m4 nor BSD m4 do a good job at following emacs regular expression syntax. `/s` working with gnu m4 appears to be undocumented and after reading some gnu m4 code and autoconf code, I don't see why it works. In any case, I found that replacing `\s` with `[  ]` (space & tab chars) works for both. I removed the commit and committed a new version with that (used a force push for that so you will need a force pull to get it). Could you check it on FreeBSD? It should work since FreeBSD uses gnu m4 too.

Reply:
This now works on FreeBSD.  Thanks!